src/System.Management.Automation/engine/remoting/server/serverremotesession.cs

```diff
@@ -813,15 +811,6 @@ private void HandleCreateRunspacePool(object sender, RemoteDataEventArgs createR
                     if (value.ContainsKey("WSManStackVersion"))
                     {
                         var wsmanStackVersion = PSObject.Base(value["WSManStackVersion"]) as Version;
```

Comment:
It seems that `WSManStackVersion` is only used within this `if` statement, so shouldn't we delete lines 813 through 825?

Reply:
Done